hydra-node/test/Hydra/Model/MockChain.hs

```diff
@@ -3,7 +3,7 @@
 
 module Hydra.Model.MockChain where
 
-import Hydra.Cardano.Api
+import Hydra.Cardano.Api hiding (Network)
```

Comment:
How come this is the only change in this file?

Reply:
We now have two things called Network. One from Hydra.Network.